internal/cmd/ls.go

```diff
@@ -0,0 +1,83 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/charmbracelet/lipgloss"
+	"strconv"
+
+	"github.com/charmbracelet/bubbles/table"
+	tea "github.com/charmbracelet/bubbletea"
+	"github.com/keygen-sh/keygen-relay/internal/common"
+	"github.com/keygen-sh/keygen-relay/internal/licenses"
+	"github.com/spf13/cobra"
+)
+
+func LsCmd(manager licenses.Manager) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "ls",
+		Short: "Print the local relay server's license pool, with stats for each license",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			licensesList, err := manager.ListLicenses(cmd.Context())
+			if err != nil {
+				return err
+			}
+
+			if len(licensesList) == 0 {
+				fmt.Fprintf(cmd.OutOrStdout(), "No licenses found.")
+
+				return nil
+			}
+
+			columns := []table.Column{
+				{Title: "ID", Width: 36},
+				{Title: "Key", Width: 50},
+				{Title: "Claims", Width: 6},
+				{Title: "NodeID", Width: 8},
```

Comment:
Can we update these columns to match what `stat` currently prints?

Reply:
done!

Now, in LS and Stat commands, there will be the next columns:
`ID`, `Claims`, `NodeID`, `Last Claimed At` and `Last Released At`